app/helpers/application_helper/button/view.rb

```diff
@@ -1,5 +1,11 @@
 class ApplicationHelper::Button::View < ApplicationHelper::Button::ButtonWithoutRbacCheck
+  # tree view GTL button is only visible on special screens
+  GTL_VIEW_TREE_BUTTON = [
```

Comment:
Yeah, please don't do this. Somehow we should look back to that DSL I mentioned above, similarly to the toolbars.
But thinking this through, do we really need this test? Would this show up on other screens?